bookwyrm/templates/snippets/fav_button.html

```diff
@@ -3,18 +3,17 @@
 {% with status.id|uuid as uuid %}
 <form name="favorite" action="/favorite/{{ status.id }}" method="POST" class="interaction fav-{{ status.id }}-{{ uuid }} {% if request.user|liked:status %}is-hidden{% endif %}" data-id="fav-{{ status.id }}-{{ uuid }}">
     {% csrf_token %}
-    <button class="button is-small" type="submit">
+    <button class="button is-small is-white" type="submit">
         <span class="icon icon-heart" title="{% trans 'Like status' %}">
-            <span class="is-sr-only">{% trans "Like status" %}</span>
         </span>
+        <span>{% trans "Like status" %}</span>
```

Comment:
Seeing this displayed, it feels more consistent to just have this read "Like" rather than "Like status", and ditto for "Boost". I think it's sufficiently clear from context that it refers to the status?